app/assets/stylesheets/application.css.scss

```diff
@@ -130,3 +130,6 @@ span.unit{
   background: rgba(0, 0, 0, 0);
 }
 
+.table > thead > tr > th, .table > thead > tr > td, .table > tbody > tr > th, .table > tbody > tr > td, .table > tfoot > tr > th, .table > tfoot > tr > td{
```

Comment:
To jest pożyczone z bootstrapa - chciałem zoverridować dokładnie ten styl, więc zamiast szukać rozwiązań, w których skompletuję odpowiedni zestaw klas o większym specifity, to przykryłem ich deklarację ;)
Działa.